enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -33,6 +37,74 @@ pub struct Handler<'a> {
     ctx: &'a Context,
 }
 
+impl<'a> Print<str> for Handler<'a> {
+    fn print(&mut self, data: &str) {
+        const STDERR: u64 = 2;
```

Comment:
Should this fd number be put in `nolibc`?

Reply:
Probably. I want a debt bug to remove `nolibc` altogether. But I think this will require `cargo make`.

Reply:
Yeah, I understand the trepidation about adding to `nolibc` when we want to rip it out. On the bright side, if we add this to `nolibc`, the compiler will remind us to update the call sites when the time comes to remove `nolibc`.

Reply:
Fixed.